content/spin/api-guides-overview.md

```diff
@@ -0,0 +1,20 @@
+title = "API Support Overview"
+template = "spin_main"
+date = "2023-03-03T03:03:03Z"
+
+---
+
+The following table shows the status of the interfaces Spin provides to applications.
+
+| Host Capabilities/Interfaces           | Stability  |    Cloud    |
+|----------------------------------------|----------|-------|
+| [HTTP Trigger](https://developer.fermyon.com/spin/http-trigger)                          | Stable   | Yes   |
+| [Redis Trigger](https://developer.fermyon.com/spin/redis-trigger)                         | Stable   | No  |
+| [Outbound HTTP](https://developer.fermyon.com/spin/http-outbound)                          | Stable   | Yes   |
+| [Outbound Redis](https://developer.fermyon.com/spin/redis-outbound)                         | Stable  | Yes   |
+| [Dynamic Configuration](https://developer.fermyon.com/spin/dynamic-configuration)                         | Experimental | No |
+| [PostgreSQL](https://developer.fermyon.com/spin/rdbms-storage)                             | Experimental | Yes |
+| [MySQL](https://developer.fermyon.com/spin/rdbms-storage)                                  | Experimental | Yes |
+| [Key-value Storage](https://developer.fermyon.com/spin/kv-store.md)                      | Stabilizing | No |
```

Comment:
I thought this was stable (as a Spin feature, albeit not on Cloud) - have put out a query to K/V folks.